src/test/kotlin/platform/mixin/MixinDuplicateTargetInspectionTest.kt

```diff
@@ -0,0 +1,350 @@
+/*
+ * Minecraft Development for IntelliJ
+ *
+ * https://mcdev.io/
+ *
+ * Copyright (C) 2024 minecraft-dev
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, version 3.0 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.demonwav.mcdev.platform.mixin
+
+import com.demonwav.mcdev.framework.EdtInterceptor
+import com.demonwav.mcdev.platform.mixin.inspection.MixinDuplicateTargetInspection
+import org.intellij.lang.annotations.Language
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(EdtInterceptor::class)
+@DisplayName("Mixin Duplicate Target Inspection Test")
+class MixinDuplicateTargetInspectionTest : BaseMixinTest() {
+
+    private fun doTest(@Language("JAVA") code: String) {
+        buildProject {
+            dir("test") {
+                java("TestMixin.java", code)
+            }
+        }
+
+        fixture.enableInspections(MixinDuplicateTargetInspection::class)
+        fixture.checkHighlighting(false, false, true)
```

Comment:
You can add one last `true` argument to remove the undesired unresolved highlights.